test/testdata/packager/lsp_features/bart/bart.rb

```diff
@@ -2,12 +2,12 @@
 
     CatchPhrase = "Don't have a cow, man."
 #   ^^^^^^^^^^^ def: catchphrase
-#   ^^^^^^^^^^^ symbol-search: "Catchphrase", name = "Bart::CatchPhrase", container = "Bart"
+#   ^^^^^^^^^^^ symbol-search: "Catchphrase", name = "CatchPhrase", container = "Bart"
 
 # Character class description
 class Character
     # ^^^^^^^^^ def: character
-    # ^^^^^^^^^ symbol-search: "Character", name = "Bart::Character", container = "Bart"
+    # ^^^^^^^^^ symbol-search: "Character", name = "Character", container = "Bart"
```

Comment:
Ditto for this one.

Reply:
Updated to unified package output in b0f62b9da
This had some other changes, but it appears those are for the best as well (hover will also now show `Bart::Character` instead of Character` which is inline with what happens for non-packages: 
![image](https://user-images.githubusercontent.com/8882905/95361226-4ff21280-0881-11eb-818a-54b75156dc8e.png)